### ISSUE TITLE
Should be able to translate names of months and days like I18n.localize

### ISSUE THREAD

**Issue by Startouf** [basecamp/local_time#57]
The support for time formats works well, but it doesn't seem to translate the names of the months, etc.

```
I18n.with_locale(:fr) do
  <%= local_time model.from, :date %> 
  # => "Saturday 28 May 2016" instead of "Samedi 28 Mai 2016"
end
```

**Comment by ACPK** [comment]
+1

**Comment by delicious-roasted-turkey** [comment]
See #22

**Comment by foucist** [comment]
This should be closed since i18n support was added in #55
